examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3997,7 +3999,21 @@ class GLTFParser {
 
 			parser.associations.get( node ).nodes = nodeIndex;
 
-			return node;
+            if ( nodeDef.skin === undefined ) return node;
+
+			return parser.getDependency( 'skin', nodeDef.skin ).then( function ( skeleton ) {
+
+				node.traverse( function ( mesh ) {
+
+					if ( ! mesh.isSkinnedMesh ) return;
+
+					mesh.bind( skeleton, _identityMatrix );
```

Comment:
For reviewers: I think the second argument should be identity matrix to mach the spec as I mentioned https://github.com/mrdoob/three.js/issues/24772#issuecomment-1325637871 Explicitly passing `_identityMatrix` may be less confusing than passing `mesh.matrixWorld` that is identity.

Reply:
Agreed, thanks!